README.md

```diff
@@ -41,6 +41,7 @@ ch_cookie_consent:
 ### Step 4: Configure to your needs
 Configure your Cookie Consent with the following possible settings
 ```yaml
+# app/config.yml
```

Comment:
For newer Symfony versions a file should automatically be created in config/packages/ch_cookie_consent.yaml, so in those cases it should not be in app/config.yml. So I rather see this be 
```
config/packages/ch_cookie_consent.yaml
```